### PR TITLE
Feature add skeleton for webapp

### PR DESCRIPTION
**Pull request by dalekatwork**
- Added base layout. We only need one layout here. It has sidebar, header and footer.
- Updated default css file to latest ant-design master https://github.com/ant-design/ant-design/commit/713949325843adaeb96b9cc7d26e5ef50821cd52

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "antd": "^4.1.5",
+    "antd": "^4.2.0",
```
Comment (robert-zaremba) [inline webapp/package.json:6]: :+1:

File: webapp/src/styles/themes/customization.less [hunk 2]
```diff
@@ -3,34 +3,42 @@
    Put here only overwrites of variables defined in default.less
 ************************************************************************/
 
-// -------- custom theme colors --------
+// custom theme colors
+@invelinq-base-color: @cyan-7;
 
+// override base colors
+@primary-color: @invelinq-base-color;
+@primary-5: @invelinq-base-color;
```
Comment (robert-zaremba) [inline webapp/src/styles/themes/customization.less]: Please, don't copy Invelinq colours and Invelinq name. Let's use something different. Can be more in an "unicorn" style.

**Review by robert-zaremba — APPROVED** [review]
Please solve all comments before merging.

File: webapp/src/styles/themes/customization.less (hunk 2 above)
Comment (dalekatwork) [reply webapp/src/styles/themes/customization.less]: done in commit 83b486a0290876e784513c81a51175bd83b5aca7

**Pull request merged**